catalogue/webapp/pages/work.js

```diff
@@ -413,7 +413,7 @@ export const WorkPage = ({
                     ])}>
                       <Icon name='underConstruction' extraClasses='margin-right-s2' />
                       <p className={`${font({s: 'HNL5', m: 'HNL4'})} no-margin`}>
-                        We’re improving the information on this page. <a href='/progress'>Find out more</a>.
+                        We’re improving the information on this page. <a href='/works/progress'>Find out more</a>.
```

Comment:
Were the progress notes supposed to be specific to works? I'm not sure that they were (even though they obviously are for the time being).

Reply:
Let's say that these progress notes are, we can generify them if we move in that direction.